packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -147,4 +147,8 @@ export function setItemInLocalStorage(key, item) {
   if (item) {
     localStorage.setItem(key, JSON.stringify(item));
   }
+}
+
+export function regexEncode(value) {
```

Comment:
maybe a comment to describe the purpose of that function? Nobody will remember what it does in like 2 weeks :P